src/components/Meter.tsx

```diff
@@ -15,7 +15,7 @@ const Meter: FC<Props> = ({
   hoverText,
 }: Props) => {
   return (
-    <>
+    <div className="p-meter-container">
```

Comment:
Ideally we avoid those wrapper classes and find a way to style without having to add markup just for layouting.

Reply:
Can we undo the changes here and in the `meter.scss` file? I assume with the two column solution most of it is not needed.